digdag-standards/src/test/java/io/digdag/standards/operator/td/TdOperatorFactoryTest.java

```diff
@@ -124,4 +124,38 @@ public void rejectQueryFileOutsideOfProjectPath()
         newOperatorFactory(TdOperatorFactory.class)
             .newOperator(newContext(projectPath, newTaskRequest().withConfig(config)));
     }
+
+    @Test
+    public void rejectResultSettingsWithoutResultConnection()
+            throws Exception
+    {
+        Path projectPath = Paths.get("").normalize().toAbsolutePath();
+
+        Config config = newConfig()
+            .set("database", "testdb")
+            .set("query", "select 1")
+            .set("result_settings", "{\"type\":\"http\"}");
+
+        exception.expectMessage("result_settings is valid only if result_connection is set");
+
+        newOperatorFactory(TdOperatorFactory.class)
+            .newOperator(newContext(projectPath, newTaskRequest().withConfig(config)));
+    }
+
+    @Test
+    public void rejectTooLargeConnectionId()
+            throws Exception
+    {
+        Path projectPath = Paths.get("").normalize().toAbsolutePath();
+
+        Config config = newConfig()
+            .set("database", "testdb")
+            .set("query", "select 1")
+            .set("result_connection", "18446744073709551616");
+
+        exception.expectMessage("for key 'result_connection' but got \"18446744073709551616\"");
```

Comment:
This message seems ambiguous to me although I know the changes of this pull request don't include the error message...